core/src/interface/ICoreContract.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import {IExtensionTypes} from "./IExtensionTypes.sol";
+
+interface ICoreContract is IExtensionTypes {
+    /*//////////////////////////////////////////////////////////////
+                                STRUCTS
+    //////////////////////////////////////////////////////////////*/
+
+    struct CallbackFunction {
+        uint256 callbackFunctionBitflag;
+        string callbackFunctionSignature;
+        address extensionContractImplementation;
+    }
+
+    struct InstalledExtension {
+        address extensionContract;
+        ExtensionFunction[] extensionABI;
```

Comment:
`extension` seems redundant since we're defining this in InstalledExtension struct?